pkgs/by-name/un/unimatrix/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, git
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "unimatrix";
+  version = "unstable-2023-04-25";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "will8211";
+    repo = pname;
+    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
+    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
+  };
+
+ installPhase = ''
```

Just an indentation fix:

```suggestion
  installPhase = ''
```